README.md

```diff
@@ -1,2 +1,111 @@
-# SIGPAC-Tools
-SIGPAC-Tools is a custom Python library that provides the user a comfortable client to retrieve data from SIGPAC api, saving the user from downloading and storing heavy files.
+# [SIGPAC-Tools](https://github.com/KhaosResearch/sigpac-tools)
+
+![Python >=3.10](https://img.shields.io/badge/python-%3E=3.10-blue.svg)
```

Comment:
Look at this version from the `eidos` package, which searches the pyproject.toml file for the version so it updates automatically.

https://github.com/KhaosResearch/eidos/blob/master/README.md?plain=1#L4